app/views/studios/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>All Studios</h1>
+
+<% @studios.each do |studio| %>
+  <div id="studio-<%= studio.id %>">
+    <h2> Studio: <%= studio.name %> </h2>
+    <b> Location: </b><%= studio.location %> </br>
+    <b> Movies: </b>
+    <ul> <% studio.movies.each do |movie| %> 
+      Title: <%= movie.title %> </br>
+      Year: <%= movie.creation_year %> </br>
+      Genre: <%= movie.genre %></br> </br>
+    <% end %> </ul>
+  </div>
+<% end %>
```

Comment:
Please don't forget that this is an HTML page and all of the text in this file should be wrapped in some form of HTML.

If you are going to use a list `<ul>` then each element, Title, Year, Genre, should be an list item `<li>`.